src/process_utility.c

```diff
@@ -1564,6 +1564,11 @@ process_altertable_end_subcmd(Hypertable *ht, Node *parsetree, ObjectAddress *ob
 		case AT_ClusterOn:
 			process_altertable_clusteron_end(ht, cmd);
 			break;
+		case AT_SetUnLogged:
+			ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+					 errmsg("hypertables cannot be unlogged")));
```

Comment:
"cannot be unlogged" is a bit of an awkward clause (though it looks decent in the test case). Should we add an `errdetail` explaining that hypertables must be logged?

Reply:
Perhaps `unlogged tables cannot be converted to hypertables`